apps/native-ui-storybook/components/List/List.docs.mdx

```diff
@@ -0,0 +1,660 @@
+            <ListItemIcon as={GasMediumIcon} color="$apple800" />
+          </Box>
+        </ListItemLeadingContent>
+        <ListItemContent>
+          <HStack justifyContent="space-between">
+            <ListItemText>Electricity</ListItemText>
+            <Badge colorScheme="green" size="small">
+              <BadgeText>Smart meter</BadgeText>
+            </Badge>
+          </HStack>
+          <ListItemSupportingText>Last reading 23/03/24</ListItemSupportingText>
+        </ListItemContent>
+        <ListItemTrailingContent>
+          <ListItemTrailingIcon as={ChevronRight01MediumIcon} />
+        </ListItemTrailingContent>
+      </ListItem>
+    </List>
+  </NativeUIProvider>
+</Canvas>
+
+```tsx
+import {
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemLeadingContent,
+  ListItemSupportingText,
+  ListItemText,
+  ListItemTrailingContent,
+  ListItemTrailingIcon,
+  ListItemContent,
+  Badge,
+  BadgeText,
+  Box,
+  HStack,
+} from '@utilitywarehouse/native-ui';
+import { ElectricityMediumIcon, GasMediumIcon } from '@utilitywarehouse/react-native-icons';
+
+const MyComponent = () => (
+  <List>
+    <ListItem onPress={() => console.log('pressed')} divider>
+      <ListItemLeadingContent>
+        <Box padding="$3" borderRadius="$lg" bg="$apple50">
+          <ListItemIcon as={ElectricityMediumIcon} color="$apple800" />
+        </Box>
+      </ListItemLeadingContent>
+      <ListItemContent>
+        <ListItemText>Electricity</ListItemText>
+        <Badge colorScheme="cyan" size="small">
+          <BadgeText>Text</BadgeText>
+        </Badge>
+        <ListItemSupportingText>Last reading 23/03/24</ListItemSupportingText>
+      </ListItemContent>
+      <ListItemTrailingContent>
+        <ListItemTrailingIcon as={ChevronRight01MediumIcon} />
+      </ListItemTrailingContent>
+    </ListItem>
+    <ListItem onPress={() => console.log('pressed')} divider={false}>
+      <ListItemLeadingContent>
+        <Box padding="$3" borderRadius="$lg" bg="$apple50">
+          <ListItemIcon as={GasMediumIcon} color="$apple800" />
+        </Box>
+      </ListItemLeadingContent>
+      <ListItemContent>
+        <HStack justifyContent="space-between">
+          <ListItemText>Electricity</ListItemText>
+          <Badge colorScheme="green" size="small">
+            <BadgeText>Smart meter</BadgeText>
+          </Badge>
+        </HStack>
+        <ListItemSupportingText>Last reading 23/03/24</ListItemSupportingText>
+      </ListItemContent>
+      <ListItemTrailingContent>
+        <ListItemTrailingIcon as={ChevronRight01MediumIcon} />
+      </ListItemTrailingContent>
+    </ListItem>
+  </List>
+);
+```
+
+### Checbox List
+
+You can also use the list to display a list of checkboxes. You can achieve this by using the `Checkbox` component as the `trailingContent` of the `ListItem`.
+
+<Canvas>
+  <NativeUIProvider>
+    <CheckboxGroup aria-label="Checkbox Group" nativeID="checkbox-group">
+      <List>
+        <ListItem
```

Comment:
This whole List Item should be preassable whiile we tap on it. This means that tapping on the text of the List item should check our uncheck the checkbox

![Screenshot 2024-08-15 at 16 19 02](https://github.com/user-attachments/assets/c114fc43-3937-4365-a48d-0e825b3a181c)

Reply:
Additionally: can you please take a look at the svg on Checkbox? [in the figma component is white. ](https://www.figma.com/design/3RY3OvLA88yZksRjOfjQJm/UW-App-UI?node-id=4454-2156)

![Screenshot 2024-08-15 at 16 21 03](https://github.com/user-attachments/assets/e599899c-bbcd-4824-ae56-9c64a12d2464)